### PR TITLE
Adds bed, bedsheets and cloaks.

### PR DESCRIPTION
**Pull request by SweptWasTaken**
Yes you can wear the bedsheets.

So basically we don't have a neck UI slot, so i'm going to pr that a day or two from now and in that pr convert all of these things to the neck slot. For right now cloaks and bedsheets go in the backpack slot.

I'll also commit inhand sprites for the bedsheets sometime tomorrow. 

Also for some fucking reason the animated states for the cult bedsheet work but the cosmos bedsheet doesn't. I swear to god I couldn't figger it out so I just left it. I'll fix it later gtfo of my face femboy.

### CODE REVIEW

File: Resources/Prototypes/Entities/Items/Clothing/cloaks.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  abstract: true
+  name: Cloak
+  description:
+  components:
+  - type: Sprite
+    sprite: Clothing/cloak.rsi
+  - type: Icon
+    sprite: Clothing/cloak.rsi
+  - type: Clothing
+    Size: 9999
```
Comment (ComicIronic) [inline Resources/Prototypes/Entities/Items/Clothing/cloaks.yml:13]: What does this do?

Comment (PJB3005) [reply Resources/Prototypes/Entities/Items/Clothing/cloaks.yml:13]: Make it impossible to be put in backpacks presumably, but also lockers?

**Comment by PJB3005** [comment]
> So basically we don't have a neck UI slot, so i'm going to pr that a day or two from now and in that pr convert all of these things to the neck slot. For right now cloaks and bedsheets go in the backpack slot.

There's servers with a neck slot nowadays?

**Comment by ike709** [comment]
TG has a neck slot.

File: Resources/Prototypes/Entities/Buildings/furniture.yml [hunk 2]
```diff
@@ -80,3 +79,16 @@
     color: "white"
   - type: Icon
     state: wooden_chair
+
+- type: entity
+  name: Bed
```
Comment (AJCM-git) [inline Resources/Prototypes/Entities/Buildings/furniture.yml:84]: Remember that we dont use upper case for improper nouns in names anymore (this applies to the rest of the PR)

Comment (SweptWasTaken) [reply Resources/Prototypes/Entities/Buildings/furniture.yml:84]: I hate this

**Comment by SweptWasTaken** [comment]
Ready for Merge

File: Resources/Prototypes/Entities/Items/Clothing/cloaks.yml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    state: capcloak
+  - type: Icon
+    state: capcloak
+  - type: Clothing
+    HeldPrefix: capcloak
+
+- type: entity
+  parent: CloakClothing
+  id: HosCloak
+  name: security cloak
```
Comment (AJCM-git) [inline Resources/Prototypes/Entities/Items/Clothing/cloaks.yml:35]: Use an acronym or the whole name of the role

Comment (SweptWasTaken) [reply Resources/Prototypes/Entities/Items/Clothing/cloaks.yml]: image

File: Resources/Prototypes/Entities/Items/Clothing/cloaks.yml [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    state: capcloak
+  - type: Icon
+    state: capcloak
+  - type: Clothing
+    HeldPrefix: capcloak
+
+- type: entity
+  parent: CloakClothing
+  id: HosCloak
+  name: hos's cloak
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Items/Clothing/cloaks.yml:35]: Shouldn't HOS be upper case since it's an acronym? Same for QM, CE, CMO, etc...

Comment (ComicIronic) [reply Resources/Prototypes/Entities/Items/Clothing/cloaks.yml:35]: I believe the standard is actually HoS and HoP for those particular acronyms, but the rest are all uppercase.

**Pull request merged**